runtime/compiler/build/files/target/z.mk

```diff
@@ -66,6 +66,7 @@ JIT_PRODUCT_SOURCE_FILES+=\
     compiler/z/codegen/J9MemoryReference.cpp \
     compiler/z/codegen/J9S390CHelperLinkage.cpp \
     compiler/z/codegen/J9S390PrivateLinkage.cpp \
+    compiler/z/codegen/J9S390JNILinkage.cpp \
```

Comment:
I think we'll need to modify the CMake builds as well.

Reply:
I don't see CMake lists contain these `cpp` file names. It operates on directories.